src/Modem.cpp

```diff
@@ -217,6 +220,7 @@ void ModemClass::poll()
 {
   while (_uart->available()) {
     char c = _uart->read();
+    _uartMillis=millis();
```

Comment:
From section 2.2.5.1 in the AT command manual:

> The DTE should wait some time (the recommended value is at least 20 ms) after the reception of an AT
command final result code or URC before issuing a new AT command to give the DCE the opportunity to
transmit the buffered URCs. Otherwise the collision of the URCs with the subsequent AT command is
still possible.

Based on this, I think this line should be moved. I'll work on some changes shortly.